sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/training_settings.py

```diff
@@ -61,6 +74,30 @@ def __init__(
         self.ensemble_model_download_timeout = ensemble_model_download_timeout
         self.allowed_training_algorithms = allowed_training_algorithms
         self.blocked_training_algorithms = blocked_training_algorithms
+        self.training_mode = training_mode
+
+    @property
```

Comment:
Please add   `@experimental` decorator for property definition here

Reply:
I have added experimental to training_mode property. Thanks!